crates/junction-api/src/http.rs

```diff
@@ -171,6 +171,10 @@ pub struct Route {
     /// should be routed.
     #[serde(default)]
     pub rules: Vec<RouteRule>,
+
+    /// Default rate limits for this route.
+    #[serde(default)]
+    pub rate_limits: Vec<RateLimit>,
```

Comment:
I can't remember whether we decided to have a single vhost-wide rate limit or multiple. the `config.py` example has a single limit, this implies multiple.